packages/server/postgres/migrations/1644936872423_addTeamPromptResponseTable.ts

```diff
@@ -0,0 +1,72 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'Reactji') THEN
+        CREATE TYPE "Reactji" AS (id text, userId text);
+      END IF;
+
+      ALTER TYPE "DiscussionTopicTypeEnum" ADD VALUE 'teamPromptResponse';
+
+      ALTER TYPE "MeetingTypeEnum" ADD VALUE 'teamPrompt';
+
+      CREATE TABLE IF NOT EXISTS "TeamPromptResponse" (
+        "id" VARCHAR(50) PRIMARY KEY,
+        "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        "meetingId" VARCHAR(100) NOT NULL,
+        "userId" VARCHAR(100) NOT NULL,
+        "content" TEXT NOT NULL,
+        "plainTextContent" TEXT NOT NULL,
+        "reactjis" "Reactji"[] NOT NULL DEFAULT array[]::"Reactji"[],
```

Comment:
+1 suggestion: add one more column `sortOrder` as v1

Reply:
i'm happy to add it but what do you think is the use case for it?

Reply:
Could be used in response grid sorting. See [here](https://github.com/ParabolInc/parabol/issues/5965#issuecomment-1042116030). Although I haven't thought about this carefully to see if `updatedAt` can be used to achieve sorting.

Reply:
@tianrunhe thanks! considering recent discussions around `sortOrder` I guess it should be an int. feel free to correct me, thought!